content/2017/vbsdcon/williams-Using_pkgsrc_for_multiplatform_deployments_in_heterogeneous_environments.md

```diff
@@ -0,0 +1,13 @@
+---
+layout: slides
+title: "Using pkgsrc for multi-platform deployments in heterogeneous environments, Made to Measure: Network Performance Analysis in FreeBSD"
```

Comment:
I think this title is combing titles from two talks. Are you sure about this editing?

And this talk (including the attached pdf file seems a pkgsrc talk, do you want to add this talk to papers.freebsd.org?

Reply:
My bad.  Talk removed.   Thanks!